tests/manage/rgw/test_obc_quota.py

```diff
@@ -0,0 +1,80 @@
+import logging
+
+import pytest
+
+from ocs_ci.ocs.resources.objectbucket import OBC
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.bucket_utils import (
+    list_obc_objects,
+    copy_random_individual_objects,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class TestOBCQuota:
+    """
+    Test OBC Quota feature
+    """
+
+    @pytest.mark.parametrize(
+        argnames="amount,interface,quota",
+        argvalues=[
+            pytest.param(
+                *[1, "RGW-OC", {"maxObjects": "1", "maxSize": "2M"}],
+            ),
+        ],
+    )
+    def test_obc_quota(
+        self, awscli_pod_session, rgw_bucket_factory, amount, interface, quota
+    ):
+        """
+        Test OBC quota feature
+            * create OBC with some quota set
+            * check if the quota works
+            * change the quota
+            * check if the new quota works
+        """
+        bucket_name = rgw_bucket_factory(amount, interface, quota=quota)[0].name
+        logger.info("Bucket created: {}".format(bucket_name))
```

Comment:
The various MCG factories do log the creation of all resources, so I think we can omit this